src/helpers.ts

```diff
@@ -0,0 +1,12 @@
+import { GuildMember, Snowflake } from 'discord.js';
+import { Logger } from '@hunteroi/advanced-logger';
+
+
```

Comment:
small thing but trailing space here is unnecessary

Reply:
Effectivement... D'habitude je n'en mets pas, donc ça m'étonne un peu de voir ce commentaire. Je le retire de suite !